### PR TITLE
[ES] Rename `comportamiento` to `conducta`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/History_of_osu!/Mapping_and_modding_timeline/es.md [hunk 1]
```diff
@@ -265,7 +265,7 @@ Los sistemas de mapping y modding están mejorando constantemente. La **cronolog
 
 - **5/8/2016:** Se creó el [blog del Quality Assurance Team](/wiki/People/Quality_Assurance_Team/Quality_Assurance_Team_Blog).
   - El QAT estaba recibiendo críticas por estar demasiado aislado del resto de la comunidad de mapping, por lo que este blog se creó para aumentar la transparencia en la toma de decisiones y la actividad de la comunidad del QAT.
-- **26/8/2016:** El [código de conducta](/wiki/Rules/Code_of_conduct_for_modding_and_mapping) se implementó para establecer las expectativas del comportamiento en modding.
+- **26/8/2016:** El [código de conducta](/wiki/Rules/Code_of_conduct_for_modding_and_mapping) se implementó para establecer las expectativas de conducta en modding.
```
Comment (UnknownOsu) [inline wiki/History_of_osu!/Mapping_and_modding_timeline/es.md:268]:
```suggestion
- **26/8/2016:** El [código de conducta](/wiki/Rules/Code_of_conduct_for_modding_and_mapping) se implementó para establecer las expectativas de conducta en el modding.
```

File: wiki/People/Global_Moderation_Team/es.md [hunk 2]
```diff
@@ -33,14 +33,14 @@ El Global Moderation Team es responsable del bienestar del chat y del foro, y se
...
 
 ### Cómo ser considerado
 
-Los factores más importantes a considerar son el comportamiento, la actividad, la comprensión de [las reglas](/wiki/Rules) y la paciencia:
+Los factores más importantes a considerar son la conducta, la actividad, la comprensión de [las reglas](/wiki/Rules) y la paciencia:
 
-1. Demostrar buen comportamiento siendo cortés y tratando a los demás usuarios con respeto. Mantener su cuenta libre de silencios o problemas serios es esencial.
-2. Demostrar actividad en el juego participando en el chat y/o los foros. [Reporte cualquier mal comportamiento](/wiki/Reporting_bad_behaviour) y problemas a través del sistema de reportes. Responder preguntas y ayudar a otros con problemas.
+1. Demostrar buena conducta siendo cortés y tratando a los demás usuarios con respeto. Mantener su cuenta libre de silencios o problemas serios es esencial.
+2. Demostrar actividad en el juego participando en el chat y/o los foros. [Reporta cualquier mala conducta](/wiki/Reporting_bad_behaviour) y problemas a través del sistema de reportes. Responder preguntas y ayudar a otros con problemas.
```
Comment (UnknownOsu) [inline wiki/People/Global_Moderation_Team/es.md:43]:
```suggestion
2. Demostrar actividad en el juego participando en el chat y/o los foros. [Reportar cualquier mala conducta](/wiki/Reporting_bad_behaviour) y problemas a través del sistema de reportes. Responder preguntas y ayudar a otros con problemas.
```

File: wiki/Tournaments/OWC/2021/es.md [hunk 3]
```diff
@@ -501,7 +501,7 @@ La clasificación final para la fase de clasificación se puede encontrar en la
 13. Se permite el intercambio de jugadores entre partidas sin limitaciones.
 14. Los mapas no deben reutilizarse en la misma partida, excepto en las revanchas.
 15. **El tamaño mínimo del equipo para un equipo es de 6 jugadores, y el máximo es de 8.**
-16. Se espera que los jugadores mantengan el juego fluido y sin retrasos. Los retrasos excesivos en los partidos por parte de los jugadores pueden dar lugar a la aplicación de sanciones a discreción del árbitro. Interrumpir el partido con juego sucio, insultar o provocar a otros jugadores o al personal, retrasar el partido u otro comportamiento inapropiado deliberado está estrictamente prohibido y será sancionado en consecuencia.
+16. Se espera que los jugadores mantengan el juego fluido y sin retrasos. Los retrasos excesivos en los partidos por parte de los jugadores pueden dar lugar a la aplicación de sanciones a discreción del árbitro. Interrumpir el partido con juego sucio, insultar o provocar a otros jugadores o al personal, retrasar el partido u otra conducta inapropiada deliberada está estrictamente prohibida y será sancionado en consecuencia.
```
Comment (UnknownOsu) [inline wiki/Tournaments/OWC/2021/es.md:504]:
```suggestion
16. Se espera que los jugadores mantengan el juego fluido y sin retrasos. Los retrasos excesivos en los partidos por parte de los jugadores pueden dar lugar a la aplicación de sanciones a discreción del árbitro. Interrumpir el partido con juego sucio, insultar o provocar a otros jugadores o al personal, retrasar el partido u otra conducta inapropiada deliberada está estrictamente prohibida y será sancionada en consecuencia.
```

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/People/Global_Moderation_Team/es.md (hunk 2 above)
Comment (agatemosu) [reply wiki/People/Global_Moderation_Team/es.md:43]: lee toda la frase

File: wiki/Tournaments/OWC/2021/es.md (hunk 3 above)
Comment (agatemosu) [reply wiki/Tournaments/OWC/2021/es.md:504]: haré otra cosa

**Review by UnknownOsu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**